src/server/api/routers/commission.ts

```
@@ -319,7 +319,7 @@ export const commission_router = createTRPCRouter({
                 published: commission.published,
                 images: [
                     {
-                        url: get_ut_url(commission.images[0]!.ut_key)
+                        url: getUTUrl(commission.images[0]!.ut_key)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)